custom/icds_reports/static/icds_reports/js/spec/month-filter.directive.spec.js

```diff
@@ -94,7 +94,7 @@ describe('Month Filter Controller', function () {
         injectPPDController();
 
         var result = controller.getPlaceholder();
-        var expected = 'Apr-Jun 2019';
+        var expected = 'Oct-Dec 1969';
```

Comment:
does not the change looks wrong? like above today is in July so the returned quarter should be April-June only no?

Reply:
updated and reverted!